--- jmorecfg.h.orig	2015-04-18 09:46:46.916706900 +0100
+++ jmorecfg.h	2015-04-18 09:48:14.316719400 +0100
@@ -142,8 +142,10 @@
 /* INT16 must hold at least the values -32768..32767. */
 
 #ifndef XMD_H                   /* X11/xmd.h correctly defines INT16 */
+#ifndef _BASETSD_H_ /* w32api defines this in basetsd.d, avoid emacs clash */
 typedef short INT16;
 #endif
+#endif
 
 /* INT32 must hold at least signed 32-bit values. */
 
